### ISSUE TITLE
poetry publish - [TypeError] read() got an unexpected keyword argument 'raw'

### ISSUE THREAD

**Issue by jykntr** [sdispater/poetry#355]
- [X] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [X] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [X] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: macOS High Sierra 10.13.6
- **Poetry version**: 0.11.3
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: [poetry's pyproject.toml](https://github.com/sdispater/poetry/blob/master/pyproject.toml)

Extra info:
- Python 3.6 was installed from 64bit installer available at https://www.python.org/downloads/release/python-366/.  
- Poetry was originally installed with `curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python` command.
- Poetry config:
```bash
$ poetry config --list
settings.virtualenvs.create = true
settings.virtualenvs.in-project = false
settings.virtualenvs.path = "/Users/jay/Library/Caches/pypoetry/virtualenvs"
repositories.staging.url = "http://localhost:3141/root/staging/"
```

## Issue
I am attempting to `publish` packages to a devpi server.  I'm testing with a cloned version of poetry to eliminate errors with the project.  With version 0.11.3 I get the following output:
```bash
$ poetry publish -vvv -r staging -u root -p root

Publishing poetry (0.11.3) to staging

[TypeError]
read() got an unexpected keyword argument 'raw'

Exception trace:
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/_vendor/cleo/application.py in run() at line 94
   status_code = self.do_run(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/console/application.py in do_run() at line 88
   return super(Application, self).do_run(i, o)
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/_vendor/cleo/application.py in do_run() at line 197
   status_code = command.run(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/console/commands/command.py in run() at line 77
   return super(BaseCommand, self).run(i, o)
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/_vendor/cleo/commands/base_command.py in run() at line 146
   status_code = self.execute(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/_vendor/cleo/commands/command.py in execute() at line 107
   return self.handle()
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/console/commands/publish.py in handle() at line 54
   self.option("repository"), self.option("username"), self.option("password")
 /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/poetry/masonry/publishing/publisher.py in publish() at line 55
   config = config_file.read(raw=True)

publish [-r|--repository REPOSITORY] [-u|--username USERNAME] [-p|--password PASSWORD] [--build]
```

In an attempt to find a version that isn't broken I went back to poetry 0.11.2 and things sorta work but I have to run the `publish` command twice for some reason:
```bash
$ poetry self:update 0.11.2
Updating to 0.11.2
  - Getting dependencies
  - Vendorizing dependencies
  - Updating poetry

poetry (0.11.2) successfully installed!

$ rm dist/*
zsh: sure you want to delete all 2 files in /Users/jay/Dropbox/poetry/dist [yn]? y

$ poetry build
Building poetry (0.11.3)
 - Building sdist
 - Built poetry-0.11.3.tar.gz

 - Building wheel
 - Built poetry-0.11.3-py2.py3-none-any.whl

$ poetry publish -vvv -r staging -u root -p root

Publishing poetry (0.11.3) to staging

$ poetry publish -vvv -r staging -u root -p root

Publishing poetry (0.11.3) to staging
 - Uploading poetry-0.11.3-py2.py3-none-any.whl 100%
 - Uploading poetry-0.11.3.tar.gz 100%
```

Please let me know if I can provide any more data!  Thanks!

**Comment by chingc** [comment]
I'm having the same issue attempting to publish to `test.pypi.org`.  Except for the `version` specifier, my `pyproject.toml` is unchanged from when I published successfully.

**Comment by chingc** [comment]
Poetry 0.11.4 came out a few hours ago and fixes this.

**Comment by jykntr** [comment]
This was fixed in release 0.11.4 with commit 9dac5bf0ad36415592fd03d049cd8d1adedd161f.
